### DISCUSSION TITLE
Add meta tag in react

### DISCUSSION THREAD

**Discussion by hollynight0** [sr-richard-collins/React-TodayTalks-v2#6] [Q&A]
How can I add meta tag in react?

**Comment by oooblizadi** [comment]
If you want to manage the document head (including meta tags) dynamically, you can use the react-helmet library.

Installation:

bash
Copy code
npm install react-helmet
Usage:
`
import React from 'react';
import { Helmet } from 'react-helmet';

const MyComponent = () => {
  return (
    <div>
      <Helmet>
        <title>Your Page Title</title>
        <meta name="description" content="Your page description here." />
        <meta name="keywords" content="react, meta, tags" />
      </Helmet>
      <h1>Hello, World!</h1>
    </div>
  );
};

export default MyComponent;`
